packages/preview/one-liner/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "one-liner"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Marco <https://github.com/mtolk>"]
+license = "MIT"
+description = "One-liner is a package containing a helper function to fit text to the available width by adjusting the text size based upon the context, so thext fits on 1 line. This is useful in templates where you don't know the length of text that is supposed to fit in specific locations in your template."
```

Comment:
There is a typo (`thext`), and I also think this description is a bit long, it could just say something along the lines of "Automatically adjust text size to make it fit on one line".